libs/expo/betterangels/src/lib/providers/keyboardToolbar/keyboardToolbarProvider.tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactNode, useState } from 'react';
+import { KeyboardToolbar } from 'react-native-keyboard-controller';
+
+import { KeyboardToolbarContext } from './keyboardToolbarContext';
+
+type TKeyboardToolbarProvider = {
+  children: ReactNode;
+};
+
+export default function KeyboardToolbarProvider(
```

Comment:
Not sure why do we need context. Either app should have arrow or not at all.

Reply:
So I kind of made a mistake and misread the ACs at first.
I coded it as if we'd need to use it conditionally. I can easily remove the context, but if we end up wanting to show the arrows then I'm not sure we'd want to do it universally, cause when you mix inputs and checkboxes on same form, for example, then acts weird. And so I figured if we ever want to customize the toolbar in any way then we'd need context, so I kept it.
Our of scope for this, I realize, so I can remove it.